### PR TITLE
[ADD] new module donation_pos

### PR DESCRIPTION
**Pull request by DiegoParadeda**
New module that implements use of donations in point of sale

### CODE REVIEW

File: donation_pos/models/pos_order.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+from odoo import api, fields, models
+
+
+class PosOrder(models.Model):
+    _inherit = "pos.order"
+
+    contribution = fields.Float(
+        string="Contribuição",
```
Comment (alexis-via) [inline donation_pos/models/pos_order.py]: Oups !

File: donation_pos/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Donation Pos",
+    "summary": """
+        This addon connects modules donation to POS.""",
+    "version": "12.0.0.0.1",
+    "license": "AGPL-3",
+    "author": "KMEE INFORMATICA LTDA,Odoo Community Association (OCA)",
+    "website": "kmee.com.br",
```
Comment (mileo) [inline donation_pos/__manifest__.py]: ops

File: donation_pos/README.rst [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Diego Paradeda <paradeda.diego@gmail.com>
```
Comment (mileo) [inline donation_pos/README.rst]: @kmee.com.br plz

File: donation_pos/__manifest__.py (hunk 2 above)
Comment (mileo) [reply donation_pos/__manifest__.py]: https://github.com/OCA/donation/blob/14.0/donation/__manifest__.py#L12-L13

File: donation_pos/static/src/js/models.js [hunk 4]
```diff
@@ -0,0 +1,182 @@
+odoo.define('donation_pos.models', function (require) {
+
+    "use strict";
+    var core = require('web.core');
+    var models = require('point_of_sale.models');
+    // var pos_tare_tools = require('pos_tare.tools');
```
Comment (mileo) [inline donation_pos/static/src/js/models.js]: Remove

**Pull request closed without merging**